src/sentry/monitors/endpoints/monitor_ingest_checkin_attachment.py

```diff
@@ -9,17 +9,15 @@
 from sentry.api.serializers import serialize
 from sentry.models import File
 
-from .base import MonitorCheckInAttachmentPermission, MonitorCheckInEndpoint
+from .base import MonitorIngestEndpoint
 
 MAX_ATTACHMENT_SIZE = 1024 * 100  # 100kb
 
 
 @region_silo_endpoint
-class MonitorIngestCheckinAttachmentEndpoint(MonitorCheckInEndpoint):
+class MonitorIngestCheckinAttachmentEndpoint(MonitorIngestEndpoint):
     # TODO(davidenwang): Add documentation after uploading feature is complete
     private = True
-    authentication_classes = MonitorCheckInEndpoint.authentication_classes + (DSNAuthentication,)
-    permission_classes = (MonitorCheckInAttachmentPermission,)
```

Comment:
This permission set is only required for downloading attachments. Anyone can upload